data/sql_updates/post_totals_create_candidate_aggregates.sql

```diff
@@ -8,7 +8,8 @@ with totals as (
         receipts,
         disbursements,
         last_cash_on_hand_end_period,
-        last_debts_owed_by_committee
+        last_debts_owed_by_committee,
+        cast('false' as boolean) as federal_funds_flag
```

Comment:
If I'm understanding this correctly, this is just defaulting to `false` in this case because the flag is only meant for presidential candidates, right?